portal/src/main/webapp/WEB-INF/jsp/dashboard/dashboard.jsp

```diff
@@ -326,20 +326,27 @@
                     $("#submit_button").click(function(){
                         iViz.submitForm(true);
                     });
-                } else {
+                } else if (cohortIdsList.length >= 2) {
                     var study_name = "";
                     var study_description = "";
-                    _.each(_cancerStudies, function (study) {
-                        study_name += study.name + "<br />";
-                        study_description += study.description + "<br />";
-                    });
-                    $("#show_study_details").css('display','block');
-                    if (study_name.length > 150){
-                       study_name = [study_name.slice(0 , 150), '<span class="truncated">', study_name.slice(150, study_name.length - 6), '</span>'].join('');
-                    }
-                    if (study_description.length > 250){
-                        study_description = [study_description.slice(0 , 250), '<span class="truncated">', study_description.slice(250, study_description.length - 6), '</span>'].join('');
+                    var collapse_name = "";
+                    var collapse_description = "";
+
+                    study_name += _cancerStudies[0].name + "<br />" + _cancerStudies[1].name;
+                    study_description += _cancerStudies[0].description + "<br />" + _cancerStudies[1].description;
+                    
+                    if (_cancerStudies.length > 2) {
+                        for (var i = 2; i < _cancerStudies.length; i++) {
+                            collapse_name += _cancerStudies[i].name + "<br />";
+                            collapse_description += _cancerStudies[i].description + "<br />";
+                        }
+                        collapse_name = collapse_name.slice(0, collapse_name.length - 6);// cut last "<br />
```

Comment:
a cleaner way to do this is `collapse_name = _cancerStudies.slice(2).map(function(study) { return study.name; }).join("<br />")` instead of the loop

Reply:
Great! Thanks!